.drone.yml

```diff
@@ -56,4 +56,62 @@ volumes:
 - name: docker
   host:
     path: /var/run/docker.sock
+---
+kind: pipeline
+name: arm64
+
+platform:
+  os: linux
+  arch: arm64
+
+steps:
+  - name: build
+    image: rancher/dapper:v0.6.0
+    commands:
+      - dapper ci
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
+
+  - name: publish-master
+    image: plugins/gcs
+    settings:
+      acl:
+        - allUsers:READER
+      cache_control: "public,no-cache,proxy-revalidate"
+      source: dist/artifacts
+      target: releases.rancher.com/harvester-node-driver/${DRONE_BRANCH}
```

Comment:
Does this override the artifacts published in the x86 pipeline? (same questin to line 101)

Reply:
dont think so. it will just publish the driver with arm64 suffix.